community/contributing/documentation_guidelines.rst

```diff
@@ -97,43 +126,56 @@ Always begin pages with their title and a Sphinx reference name:
     ======================
 
 The reference allows linking to this page using the ``:ref:`` format, e.g.
-``:ref:`doc_insert_your_title_here``` would link to the above example page
-(note the lack of leading underscore in the reference).
+``:ref:`doc_insert_your_title_here``` would link to the above example page (note
+the lack of leading underscore in the reference).
+
+Write your titles like plain sentences, without capitalizing each word:
+
+-  **Good**: Understanding signals in Godot
+-  **Bad**: Understanding Signals In Godot
+
+Only propers nouns, projects, people, and node class names should have their
+first letter capitalized.
+
+Sphinx and reStructuredText syntax
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Check Sphinx’s `reST Primer <https://www.sphinx-doc.org/en/stable/rest.html>`__
+and the `official reference <http://docutils.sourceforge.net/rst.html>`__ for
+details on the syntax.
 
-Also, avoid American CamelCase titles: title's first word should begin
-with a capitalized letter, and every following word should not. Thus,
-this is a good example:
+Sphinx uses specific reST comments to do specific operations, like defining the
+table of contents (``:toctree:``) or cross-referencing pages. Check the
+`official Sphinx documentation
+<https://www.sphinx-doc.org/en/stable/index.html>`__ for more details, or see
+how things are done in existing pages and adapt it to your needs.
```

Comment:
It could be useful to reference https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html directly for documentation on directives like `.. seealso::` or `.. note::`.